src/display/network.js

```diff
@@ -593,8 +569,6 @@ PDFNetworkStreamRangeRequestReader.prototype = {
   },
 };
 
-setPDFNetworkStreamClass(PDFNetworkStream);
```

Comment:
webpack can package node_stream and network in one file and initialize these in some random order. so it's better to call setPDFNetworkStreamClass from api.js and remove this line from the network.js